networking/SimpleTileEntities.md

```diff
@@ -0,0 +1,58 @@
+Simple Tile Entity Server to Client Networking
+=============================
+
+If you want to keep the client side tile entity synced with the server, you do not need the full networking system
+
+You can split the nbt saving/loading for the syncable items into their own functions
+
+    @Override
+    public void writeToNBT(NBTTagCompound tag)
+    {
+        super.writeToNBT(tag);
+        writeCustomNBT(tag);
+    }
+    
+    public void writeCustomNBT(tag)
```

Comment:
Why the redirection? No explanation and just complicates things you could just call read/write directly.